mbq/atomiq/test_utils.py

```diff
@@ -0,0 +1,38 @@
+from . import constants, models
+
+
+def celery_publish_call(task, *args, **kwargs):
+    tasks = models.CeleryTask.objects.filter(
+        state=constants.TaskStates.ENQUEUED,
+        task_name=task.name,
+    )
+    if args or kwargs:
+        tasks = tasks.filter(
+            task_arguments={
+                'args': args,
+                'kwargs': kwargs,
+            }
+        )
+    return tasks.exists()
```

Comment:
If you only pass in the task, this will just check that the task name matches. If you pass in args & kwargs, it will also make sure the args and kwargs match. Dis OK?

Reply:
Didn't we have a long conversation on Friday about how this could easily cause flaky tests?

Reply:
It's possible that came up but I don't remember exactly. talk IRL?